tests/test_for_support/test_for_biomass.py

```diff
@@ -370,8 +566,13 @@ def test_fast_growth_default(model, boolean):
 
 
 @pytest.mark.parametrize("model, number", [
-    ("direct_met_single_compartment", 1),
-    ("precursors_producing", 0)
+    ("only_direct_mets", 3),
+    ("only_direct_mets_false_positive_candidate_EX_reactant_T_product", 3),
+    ("only_direct_mets_false_positive_candidate_EX_reactant_T_reactant", 3),
+    ("only_direct_mets_false_positive_candidate_EX_product_T_reactant", 3),
+    ("only_direct_mets_false_positive_candidate_EX_product_T_product", 3),
+    ("direct_mets_with_false_positive", 2),
+    ("precursors_producing", 0),
```

Comment:
Very extensive tests, I like it :+1:

Reply:
I had to put many of these in to pass the Codecov tests